### DISCUSSION TITLE
ccf-validation-tools/Blood_vasculature/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#235] [General]
# ccf-validation-tools/Blood_vasculature/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Blood_vasculature/comments/

**Comment by emquardokus** [comment]
5-17-2023 Follow-up needed to add more relationships to this VCCF-ontology: https://github.com/obophenotype/vccf-ontology

**Comment by aleixpuigb** [comment]
There are 55 new vasculature terms that need to be added to the blood vasculature table:

Blood Vasculature : segmental pulmonary vein (UBERON:8600051)
Blood Vasculature : subsegmental pulmonary vein (UBERON:8600052)
Blood Vasculature : esophageal vein (UBERON:8600053)
Blood Vasculature : supreme intercostal vein (UBERON:8600054)
Blood Vasculature : atrioventricular nodal artery (UBERON:8600055)
Blood Vasculature : lateral sacral vein (UBERON:8600056)
Blood Vasculature : middle hemorrhoidal vein (UBERON:8600057)
Blood Vasculature : uterine vein (UBERON:8600058)
Blood Vasculature : deep cervical vein (UBERON:8600059)
Blood Vasculature : subsegmental pulmonary artery (UBERON:8600060)
Blood Vasculature : intra-acinar venule (UBERON:8600061)
Blood Vasculature : thyrocervical trunk (UBERON:8600062)
Blood Vasculature : septal perforating artery (UBERON:8600063)
Blood Vasculature : posterolateral artery (UBERON:8600064)
Blood Vasculature : lateral femoral circumflex vein (UBERON:8600065)
Blood Vasculature : basivertebral vein (UBERON:8600066)
Blood Vasculature : bronchial venule (UBERON:8600067)
Blood Vasculature : mental vein (UBERON:8600068)
Blood Vasculature : deep lingual vein (UBERON:8600069)
Blood Vasculature : dorsal lingual vein (UBERON:8600070)
Blood Vasculature : sublingual vein (UBERON:8600071)
Blood Vasculature : glandular venous plexus (UBERON:8600072)
Blood Vasculature : circumflex humeral vein (UBERON:8600073)
Blood Vasculature : thoracoacromial vein (UBERON:8600074)
Blood Vasculature : transverse cervical vein (UBERON:8600075)
Blood Vasculature : suboccipital venous plexus (UBERON:8600076)
Blood Vasculature : pulmonary arteriole (UBERON:8600077)
Blood Vasculature : ascending pharyngeal artery (UBERON:8600078)
Blood Vasculature : dural artery (UBERON:8600079)
Blood Vasculature : pericardial artery (UBERON:8600080)
Blood Vasculature : percardiacophrenic artery (UBERON:8600081)
Blood Vasculature : superficial iliac circumflex vein (UBERON:8600082)
Blood Vasculature : fibular vein (UBERON:8600083) (synonym of: set of peroneal veins)
Blood Vasculature : prostatic venule (UBERON:8600084)
Blood Vasculature : hepatic portal venule (UBERON:8600085)
Blood Vasculature : incisor artery (UBERON:8600086)
Blood Vasculature : external nasal vein (UBERON:8600087)
Blood Vasculature : supratrochlear vein (UBERON:8600088)
Blood Vasculature : transverse nasal root vein (UBERON:8600089)
Blood Vasculature : hypophyseal vein (UBERON:8600090)
Blood Vasculature : inferior anastomotic vein (UBERON:8600091)
Blood Vasculature : radial vein (UBERON:8600092)
Blood Vasculature : ulnar vein (UBERON:8600093)
Blood Vasculature : intra-acinar arteriole (UBERON:8600094)
Blood Vasculature : ascending palatine artery (UBERON:8600095)
Blood Vasculature : submental artery (UBERON:8600096)
Blood Vasculature : deep lingual artery (UBERON:8600097)
Blood Vasculature : dorsal lingual artery (UBERON:8600098)
Blood Vasculature : accessory meningeal artery (UBERON:8600099)
Blood Vasculature : artery of pterygoid canal (UBERON:8600100)
Blood Vasculature : descending palatine artery (UBERON:8600101)
Blood Vasculature : left anterior descending artery (UBERON:8600102)
Blood Vasculature : anterior internal vertebral venous plexus (UBERON:8600103)
Blood Vasculature : intervertebral vein (UBERON:8600104)
Blood Vasculature : lumbar vein (UBERON:8600105)

**Reply by emquardokus** [reply]
thank-you! I'll send these to Griffin to update his database for the next release version table. This is GREAT!

**Comment by emquardokus** [comment]
Sent email to Griffin Weber Cc Katy, Aleix to add 55 new terms to VCCF database and Blood vasculature v1.7 DRAFT table for October.

**Reply by aleixpuigb** [reply]
Thank you!

**Comment by aleixpuigb** [comment]
New term in release v2024-07-10 (non-HuBMAP funded):

squamous endothelial cell of venule (CL:0008040)

**Comment by aleixpuigb** [comment]
**New terms for Blood vasculature:**

Blood Vasculature : anterior humeral circumflex artery (UBERON:8600106)
Blood Vasculature : posterior humeral circumflex artery (UBERON:8600107)
Blood Vasculature : ethmoidal vein (UBERON:8420001)
Blood Vasculature : anterior ethmoidal vein (UBERON:8420002)
Blood Vasculature : posterior ethmoidal vein (UBERON:8420003)

**Comment by emquardokus** [comment]
**Blood_Vasculature_v1.7**
https://docs.google.com/spreadsheets/d/1xNqlM7EJMa7wVodxtcIJBI9kx7WfPTGCO6_dzdJPi4I/edit?gid=1958526503#gid=1958526503

**Different labels**
1. 2 instances fixed to ascending vasa recta.-this was a term updated in the ontology recently. The term UBERON:0009091 has a different name/label in the source ontology in the following 4 rows 2945, 2945, 2946, 2946. The name/label in the ASCT+B table is vasa recta ascending limb and the one in the ontology is ascending vasa recta. For reference, the given name/label by SMEs is ascending vasa recta of kidney. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2. 4 instances fixed to descending vasa recta--this was recently updated in Uberon. The term UBERON:0009202 has a different name/label in the source ontology in the following 8 rows 1631, 1631, 1632, 1632, 1633, 1633, 1634, 1634. The name/label in the ASCT+B table is vasa recta descending limb and the one in the ontology is descending vasa recta. For reference, the given name/label by SMEs is descending vasa recta of kidney. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

3. 372 instances fixed to inferior vena cava. The term UBERON:0001072 has a different name/label in the source ontology in the following 744 rows 2599, 2599, 2600, 2600, 2601, 2601, 2602, 2602, 2603, 2603, 2604, 2604, 2605, 2605, 2606, 2606, 2607, 2607, 2608, 2608, 2609, 2609, 2610, 2610, 2611, 2611, 2612, 2612, 2613, 2613, 2614, 2614, 2615, 2615, 2616, 2616, 2617, 2617, 2618, 2618, 2619, 2619, 2620, 2620, 2621, 2621, 2622, 2622, 2623, 2623, 2624, 2624, 2625, 2625, 2626, 2626, 2627, 2627, 2628, 2628, 2629, 2629, 2630, 2630, 2631, 2631, 2632, 2632, 2633, 2633, 2634, 2634, 2635, 2635, 2636, 2636, 2637, 2637, 2638, 2638, 2639, 2639, 2640, 2640, 2641, 2641, 2642, 2642, 2643, 2643, 2644, 2644, 2645, 2645, 2646, 2646, 2647, 2647, 2648, 2648, 2649, 2649, 2650, 2650, 2651, 2651, 2652, 2652, 2653, 2653, 2654, 2654, 2655, 2655, 2656, 2656, 2657, 2657, 2658, 2658, 2659, 2659, 2660, 2660, 2661, 2661, 2662, 2662, 2663, 2663, 2664, 2664, 2665, 2665, 2666, 2666, 2667, 2667, 2668, 2668, 2669, 2669, 2670, 2670, 2671, 2671, 2672, 2672, 2673, 2673, 2674, 2674, 2675, 2675, 2676, 2676, 2677, 2677, 2678, 2678, 2679, 2679, 2680, 2680, 2681, 2681, 2682, 2682, 2683, 2683, 2684, 2684, 2685, 2685, 2686, 2686, 2687, 2687, 2688, 2688, 2689, 2689, 2690, 2690, 2691, 2691, 2692, 2692, 2693, 2693, 2694, 2694, 2695, 2695, 2696, 2696, 2697, 2697, 2698, 2698, 2699, 2699, 2700, 2700, 2701, 2701, 2702, 2702, 2703, 2703, 2704, 2704, 2705, 2705, 2706, 2706, 2707, 2707, 2708, 2708, 2709, 2709, 2710, 2710, 2711, 2711, 2712, 2712, 2713, 2713, 2714, 2714, 2715, 2715, 2716, 2716, 2717, 2717, 2718, 2718, 2719, 2719, 2720, 2720, 2721, 2721, 2722, 2722, 2723, 2723, 2724, 2724, 2725, 2725, 2726, 2726, 2727, 2727, 2728, 2728, 2729, 2729, 2730, 2730, 2731, 2731, 2732, 2732, 2733, 2733, 2734, 2734, 2735, 2735, 2736, 2736, 2737, 2737, 2738, 2738, 2739, 2739, 2740, 2740, 2741, 2741, 2742, 2742, 2743, 2743, 2744, 2744, 2745, 2745, 2746, 2746, 2747, 2747, 2748, 2748, 2749, 2749, 2750, 2750, 2751, 2751, 2752, 2752, 2753, 2753, 2754, 2754, 2755, 2755, 2756, 2756, 2757, 2757, 2758, 2758, 2759, 2759, 2760, 2760, 2761, 2761, 2762, 2762, 2763, 2763, 2764, 2764, 2765, 2765, 2766, 2766, 2767, 2767, 2768, 2768, 2769, 2769, 2770, 2770, 2771, 2771, 2772, 2772, 2773, 2773, 2774, 2774, 2775, 2775, 2776, 2776, 2777, 2777, 2778, 2778, 2779, 2779, 2780, 2780, 2781, 2781, 2782, 2782, 2783, 2783, 2784, 2784, 2785, 2785, 2786, 2786, 2787, 2787, 2788, 2788, 2789, 2789, 2790, 2790, 2791, 2791, 2792, 2792, 2793, 2793, 2794, 2794, 2795, 2795, 2796, 2796, 2797, 2797, 2798, 2798, 2799, 2799, 2800, 2800, 2801, 2801, 2802, 2802, 2803, 2803, 2804, 2804, 2805, 2805, 2806, 2806, 2807, 2807, 2808, 2808, 2809, 2809, 2810, 2810, 2811, 2811, 2812, 2812, 2813, 2813, 2814, 2814, 2815, 2815, 2816, 2816, 2817, 2817, 2818, 2818, 2819, 2819, 2820, 2820, 2821, 2821, 2822, 2822, 2823, 2823, 2824, 2824, 2825, 2825, 2826, 2826, 2827, 2827, 2828, 2828, 2829, 2829, 2830, 2830, 2831, 2831, 2832, 2832, 2833, 2833, 2834, 2834, 2835, 2835, 2836, 2836, 2837, 2837, 2838, 2838, 2839, 2839, 2840, 2840, 2841, 2841, 2842, 2842, 2843, 2843, 2844, 2844, 2845, 2845, 2846, 2846, 2847, 2847, 2848, 2848, 2849, 2849, 2850, 2850, 2851, 2851, 2852, 2852, 2853, 2853, 2854, 2854, 2855, 2855, 2856, 2856, 2857, 2857, 2858, 2858, 2859, 2859, 2860, 2860, 2861, 2861, 2862, 2862, 2863, 2863, 2864, 2864, 2865, 2865, 2866, 2866, 2867, 2867, 2868, 2868, 2869, 2869, 2870, 2870, 2871, 2871, 2872, 2872, 2873, 2873, 2874, 2874, 2875, 2875, 2876, 2876, 2877, 2877, 2878, 2878, 2879, 2879, 2880, 2880, 2881, 2881, 2882, 2882, 2883, 2883, 2884, 2884, 2885, 2885, 2886, 2886, 2887, 2887, 2888, 2888, 2889, 2889, 2890, 2890, 2891, 2891, 2892, 2892, 2893, 2893, 2894, 2894, 2895, 2895, 2896, 2896, 2897, 2897, 2898, 2898, 2899, 2899, 2900, 2900, 2901, 2901, 2902, 2902, 2903, 2903, 2904, 2904, 2905, 2905, 2906, 2906, 2907, 2907, 2908, 2908, 2909, 2909, 2910, 2910, 2911, 2911, 2912, 2912, 2913, 2913, 2914, 2914, 2915, 2915, 2916, 2916, 2917, 2917, 2918, 2918, 2919, 2919, 2920, 2920, 2921, 2921, 2922, 2922, 2923, 2923, 2924, 2924, 2925, 2925, 2926, 2926, 2927, 2927, 2928, 2928, 2929, 2929, 2930, 2930, 2931, 2931, 2932, 2932, 2933, 2933, 2934, 2934, 2935, 2935, 2936, 2936, 2937, 2937, 2938, 2938, 2939, 2939, 2940, 2940, 2941, 2941, 2942, 2942, 2943, 2943, 2944, 2944, 2945, 2945, 2946, 2946, 2947, 2947, 2948, 2948, 2949, 2949, 2950, 2950, 2951, 2951, 2952, 2952, 2953, 2953, 2954, 2954, 2955, 2955, 2956, 2956, 2957, 2957, 2958, 2958, 2959, 2959, 2960, 2960, 2961, 2961, 2962, 2962, 2963, 2963, 2964, 2964, 2965, 2965, 2966, 2966, 2967, 2967, 2968, 2968, 2969, 2969, 2970, 2970. The name/label in the ASCT+B table is posterior vena cava and the one in the ontology is inferior vena cava. For reference, the given name/label by SMEs is inferior vena cava. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
